### PR TITLE
doc(openid-connect): clarify that OIDC is only supported for the Apache web server

### LINKED ISSUE #7134: Make it clearer in the documentation that OIDC login support only works on apache
The OIDC support seems to be implemented using a module in apache. This is not at all obvious directly from the documentation.

### PR DESCRIPTION
**Pull request by aftix**
Fixes: https://github.com/FreshRSS/FreshRSS/issues/7134

Closes 7134

Pull request checklist:

- [x] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [x] documentation updated

### CODE REVIEW

File: docs/en/admins/16_OpenID-Connect.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ See: [What is OpenID Connect?](https://openid.net/connect/).
 
 This is one of the [access control methods](09_AccessControl.md) supported by FreshRSS.
 
-OIDC support is provided by [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc).
+OIDC support is provided by the [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc)
+Apache module. OIDC is not supported on web servers other than Apache.
```
Comment (Alkarex) [inline docs/en/admins/16_OpenID-Connect.md]: The formulation is not exactly true. OIDC in FreshRSS is supported by any Web server that implements an OIDC module. I have not investigated whether any Web server implements it though, besides Apache.

Comment (Alkarex) [inline docs/en/admins/16_OpenID-Connect.md]:
```suggestion
Apache module. OIDC is not supported on Web servers other than Apache.
```

Comment (Alkarex) [inline docs/en/admins/16_OpenID-Connect.md]:
```suggestion
OIDC support is provided by the Apache module [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc).
This documentation is about OIDC as available in our official Docker image, or when using an Apache Web server.
```

What about this formulation, @aftix ?

**Comment by aftix** [comment]
Is the configuration for freshrss, or are the configuration settings just for the apache module and fresh rss can handle any OIDC relaying party in the web server?

**Comment by Alkarex** [comment]
FreshRSS implements HTTP Basic Auth, so anything in front that handles OIDC, other forms of single sign on, or related credential check could work, as long as the result is forwarded as HTTP Basic Auth to FreshRSS, like `mod_auth_openidc` does.

File: docs/en/admins/16_OpenID-Connect.md (hunk 1 above)
Comment (Frenzie) [reply docs/en/admins/16_OpenID-Connect.md]: Indeed, I would remove that last sentence or rephrase it as something like "this documentation only covers Apache"

**Comment by aftix** [comment]
I am currently away from my computer for the holidays, I will come back to this pr when I return and rewrite it.

File: docs/en/admins/16_OpenID-Connect.md (hunk 1 above)
Comment (Alkarex) [reply docs/en/admins/16_OpenID-Connect.md]: @Frenzie What about https://github.com/FreshRSS/FreshRSS/pull/7135#discussion_r1895586597 ?

Comment (Frenzie) [reply docs/en/admins/16_OpenID-Connect.md]: That works.

Comment (aftix) [reply docs/en/admins/16_OpenID-Connect.md]: This wording seems clear to me.

**Comment by Alkarex** [comment]
> I am currently away from my computer for the holidays, I will come back to this pr when I return and rewrite it.

All good 🎄

**Comment by Alkarex** [comment]
Help welcome:
* https://github.com/FreshRSS/FreshRSS/issues/8031

**Pull request merged**